Plugins/Flow.Launcher.Plugin.BrowserBookmark/FirefoxBookmarkLoader.cs

```diff
@@ -5,140 +5,133 @@
 using System.IO;
 using System.Linq;
 
-namespace Flow.Launcher.Plugin.BrowserBookmark
+namespace Flow.Launcher.Plugin.BrowserBookmark;
+
+public abstract class FirefoxBookmarkLoaderBase : IBookmarkLoader
 {
-    public abstract class FirefoxBookmarkLoaderBase : IBookmarkLoader
-    {
-        public abstract List<Bookmark> GetBookmarks();
+    public abstract List<Bookmark> GetBookmarks();
 
-        private const string queryAllBookmarks = @"SELECT moz_places.url, moz_bookmarks.title
-              FROM moz_places
-              INNER JOIN moz_bookmarks ON (
+    private const string QueryAllBookmarks = """
```

Comment:
Rename `private const` to start with a capital letter. Same with `DbPathFormat`.

Reply:
I feel like this add newline that does not exist before? have you tested whether it still worked?

Reply:
This does add a linebreak at the and and also reduce indenting. Luckily, SQL is not a language with significant whitespace, so I don't think it matters, and I think a raw string literal (`"""`) visually conveys the idea that the string is multiline much better than a verbatim string (`@"`). I did test if it works: I added this text when reading from the database:
![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/3993179/3905a891-cc4d-4cfc-816f-176cd109ec9c)
![image](https://github.com/Flow-Launcher/Flow.Launcher/assets/3993179/65083f34-0ab7-4ce6-a7e4-8f2ba58b4aa0)